app/services/DR/getToken.js

```diff
@@ -30,3 +32,27 @@ export default async function getToken(needNewToken = false) {
     }
   }
 }
+
+export async function getTokenMepid(needNewToken = false) {
+  const savedToken = await GetStoreData('MEPID_API_TOKEN', true);
+
+  const mepidUrl = 'https://covidrdapi.msp.gob.do:8443';
+  const mepidUsername = 'covidrd';
+  const mepidPassword = 'msp@2020app';
+
+  if (savedToken && !needNewToken) {
+    return savedToken;
+  } else {
+    try {
+      const response = await Axios.post(
+        `${mepidUrl}/api/auth?username=${mepidUsername}&password=${mepidPassword}`,
+      );
```

Comment:
you can use **fetch** because that's we mostly use, and delete the import of axios in the top of the file, and use the word `token` instead of `response.data.token`

```suggestion
      const response = await fetch( `${mepidUrl}/api/auth?username=${mepidUsername}&password=${mepidPassword}`,
        { method: 'POST' },);
        
      const { token } = await response.json();
```